apps/academy/src/pages/tracks/oracles-api3/index.tsx

```diff
@@ -0,0 +1,88 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import React from "react";
+import { TrackCard } from "ui";
+
+import Spinner from "@/components/Spinner";
+import TracksLayout from "@/components/TracksLayout";
+import { api } from "@/utils/api";
+
+const OraclesApi3TrackPage = () => {
+  const router = useRouter();
+
+  const path = router.pathname;
+
+  const { data: allLessonsData } = api.lessons.getLessonsByTrackPath.useQuery(
+    { trackPath: path },
+    {
+      refetchOnWindowFocus: false,
+    },
+  );
+  // const lessonsArray = [
```

Comment:
Hey @Markkos89 
I think it was okay to `useQuery` and fetch from the DB, and therefore no need to use the LessonArray approach - is that correct? :goldfish memory: 🤣